### DISCUSSION TITLE
Ideas and feedback for @liveblocks/vue

### DISCUSSION THREAD

**Discussion by GuillaumeSalles** [liveblocks/liveblocks#631] [Feedback]
Liveblocks does not have a specific package for Vue.js. 

Currently, all our Vue examples are using `@liveblocks/client` directly.

If you're interested in having a specific package for Vue.js or if you have an opinion about the shape of the API, please let us know by answering this issue.

Thank you!

**Comment by CTNicholas** [comment]
I don't use Vue too much, so I don't have a strong opinion on the subject matter, but if anyone is looking for some Vue 3 hooks/providers that replicate `liveblocks-react` I quickly built a few for a recent project:
https://github.com/CTNicholas/wordle-wars/tree/master/src/lib-liveblocks

`useList` is slightly different; it uses a function call because of https://github.com/liveblocks/liveblocks/issues/66

**Comment by GuillaumeSalles** [comment]
Hi @thanhvumsft, Thanks for the kinds words! 

`@liveblocks/react` is just a wrapper around `@liveblocks/client` so in theory, you should be able to do everything `@liveblocks/react` is doing. Which `@liveblocks/react` API are you lacking right now? Maybe we can provide code examples until we focus on an official `@liveblocks/vue` package.

**Comment by adigau** [comment]
Hello @thanhvumsft! First of all, thank you for taking the time to share your feedback; it's a great tribute to our mission. It also shows you understand the impact kudos can have on a company like ours and the engineers working on it daily. We shared it in our internal Slack, and it received a lot of :love: reactions haha

Second of all, we would be very curious to know more about the product you're building. I followed your link and was brought to a flow builder with tool selection. Couldn't say where the multiplayer was implemented.

@GuillaumeSalles and I would love to jump on a call to understand your use case better. If you're interested, you can send me an email so that we can schedule it together: adrien@liveblocks.io

Talk to you soon! And again, thank you for the kind words 🧡

**Comment by adigau** [comment]
That would be awesome @thanhvumsft! We'd love to learn more about your work. Thanks!

**Comment by thanhvumsft** [comment]
@GuillaumeSalles could you please remove all my comments in this issue?
https://github.com/liveblocks/liveblocks/issues/1

**Reply by stevenfabre** [reply]
We just removed your comments on that issue @thanhvumsft

**Comment by peoray** [comment]
hi @GuillaumeSalles @adigau  is this still a valid issue? I'd like to pick it up :)

**Reply by adigau** [reply]
We haven't made any progress on an official Vue package, so this is still clearly a valid issue!

**Reply by peoray** [reply]
@adigau Thank you for the response. How do I proceed? I'd love to contribute with the team's help ofc 🙂

**Reply by peoray** [reply]
@GuillaumeSalles @adigau bumping this for visibility :)

**Comment by tongesy** [comment]
A Vue package would be great; If only to simplify interacting with Liveblocks storage data using v-model. Even a guide on this would be useful.
